ticketbuyer.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"time"
+
+	"github.com/decred/dcrrpcclient"
+	"github.com/decred/dcrwallet/ticketbuyer"
+	"github.com/decred/dcrwallet/wallet"
+)
+
+const (
+	defaultTicketAddress      = ""
+	defaultMaxFee             = 1.0
+	defaultMinFee             = 0.01
+	defaultMaxPriceScale      = 2.0
+	defaultMinPriceScale      = 0.7
+	defaultPriceTarget        = 0.0
+	defaultAvgVWAPPriceDelta  = 2880
+	defaultMaxPerBlock        = 5
+	defaultHighPricePenalty   = 1.3
+	defaultTicketFeeInfo      = false
+	defaultBlocksToAvg        = 11
+	defaultFeeTargetScaling   = 1.05
+	defaultDontWaitForTickets = false
+	defaultMaxInMempool       = 40
+	defaultExpiryDelta        = 16
+)
+
+// purchaseManager is the main handler of websocket notifications to
+// pass to the purchaser and internal quit notifications.
+type purchaseManager struct {
```

Comment:
I would really like to see this not part of the main package. Any reason in particular it was added here?